.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Checks
+
+- [ ] I have read the [contributing guidelines](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/CONTRIBUTING.md).
+- [ ] If some rule is added/updated/removed, I've regenerated the rules list.
+- [ ] if some rule meta info is changed, I've regenerated the plugin shared configs.
```

Comment:
This one should start by capital letter too, right?

Reply:
```suggestion
- [ ] If some rule meta info is changed, I've regenerated the plugin shared configs.
```

Good catch!